Fabulous.XamarinForms/samples/FabulousWeather/FabulousWeather/App.fs

```diff
@@ -0,0 +1,154 @@
+﻿namespace FabulousWeather
+
+open Fabulous
+open Fabulous.XamarinForms
+open Fabulous.XamarinForms.LiveUpdate
+open Xamarin.Forms
+
+module App =
+    type Operator = Add | Subtract | Multiply | Divide 
+
+    /// Represents a calculator button press
+    type Msg =
+        | Refresh
+
+    
+    type DataItem = {Name:string;Value:int}
+    type Model = {Temp:int;Items: DataItem list}
+
+    let initial = {Temp=61;Items=[
+        {Name="Pressure";Value=10}
+        {Name="UV Index";Value=3}
+        {Name="Wind Speed";Value=0}
+        {Name="Humidity";Value=65}
+        {Name="Min Temp";Value=50}
+        {Name="Max Temp";Value=80}
+        ]}
+     
+    let update msg model =
+        match msg with
+        | Refresh -> initial
```

Comment:
Could you try to implement a really simple mechanism of calling an API to get the temperature of the day of Seattle?
Only to provide a meaningful sample of how to handle APIs and asynchronous calls, instead of just a nice looking UI.

Reply:
Got this working using open weather api.  Users would just have to put in a key, which is free to get.